docker/build/devdeps.manylinux.Dockerfile

```diff
@@ -70,10 +74,8 @@ RUN dnf install -y --nobest --setopt=install_weak_deps=False \
     && git remote add origin https://github.com/pybind/pybind11 \
     && git fetch origin --depth=1 $pybind11_commit && git reset --hard FETCH_HEAD \
     && mkdir -p /pybind11-project/build && cd /pybind11-project/build \
-    && python3 -m ensurepip --upgrade && python3 -m pip install pytest \
-    && cmake -G Ninja ../ -DCMAKE_INSTALL_PREFIX="$PYBIND11_INSTALL_PREFIX" -DPYTHON_EXECUTABLE="$(which python3)" \
+    && cmake -G Ninja ../ -DCMAKE_INSTALL_PREFIX="$PYBIND11_INSTALL_PREFIX" -DPYTHON_EXECUTABLE="$(which python3)" -DPYBIND11_TEST=False \
```

Comment:
Should we be capping the number of ninja threads the build uses? I think I've seen mystery failures, possibly if the machine runs out of resources, when there are a lot of PRs going at once.

Reply:
Maybe, but this seems pretty unrelated to this PR? The build here doesn't add any more or less build threads than we currently have.

Reply:
This comment relates to various spontaneous failures in general and not this PR per se. Using `ninja` is great until it isn't. And when it isn't, it can be completely non-scalable.